.github/workflows/ci-auto-commit-mac.yml

```diff
@@ -17,16 +17,14 @@ jobs:
         uses: actions/checkout@v2   
         with:
           ref: develop 
-      - name: Setup Python
-        run: |
-          brew update
-          brew install python@3.7
-          brew unlink python@3.9
-          brew link --force --overwrite python@3.7
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.8"
```

Comment:
This is a very useful GitHub action.

Reply:
Yes. What resources are available can be checked by looking at the tools available for each image such as here:

https://github.com/actions/runner-images/blob/main/images/macos/macos-11-Readme.md#python